libavcodec/x86/vvc_mc.asm

```diff
@@ -0,0 +1,460 @@
+
+SECTION_RODATA 64
+
+                    ; 1x, hpelIfIdx == 0, Table 27
+pw_vvc_luma_filters dw  0, 0,   0, 64,  0,   0,  0,  0,
+                    dw  0, 1,  -3, 63,  4,  -2,  1,  0,
+                    dw -1, 2,  -5, 62,  8,  -3,  1,  0,
+                    dw -1, 3,  -8, 60, 13,  -4,  1,  0,
+                    dw -1, 4, -10, 58, 17,  -5,  1,  0,
+                    dw -1, 4, -11, 52, 26,  -8,  3, -1,
+                    dw -1, 3,  -9, 47, 31, -10,  4, -1,
+                    dw -1, 4, -11, 45, 34, -10,  4, -1,
+                    dw -1, 4, -11, 40, 40, -11,  4, -1,
+                    dw -1, 4, -10, 34, 45, -11,  4, -1,
+                    dw -1, 4, -10, 31, 47,  -9,  3, -1,
+                    dw -1, 3,  -8, 26, 52, -11,  4, -1,
+                    dw  0, 1,  -5, 17, 58, -10,  4, -1,
+                    dw  0, 1,  -4, 13, 60,  -8,  3, -1,
+                    dw  0, 1,  -3,  8, 62,  -5,  2, -1,
+                    dw  0, 1,  -2,  4, 63,  -3,  1,  0,
+                
+                    ; 1x, hpelIfIdx == 1, Table 27
+                    dw  0, 0,   0, 64,  0,   0,  0,  0,
+                    dw  0, 1,  -3, 63,  4,  -2,  1,  0,
+                    dw -1, 2,  -5, 62,  8,  -3,  1,  0,
+                    dw -1, 3,  -8, 60, 13,  -4,  1,  0,
+                    dw -1, 4, -10, 58, 17,  -5,  1,  0,
+                    dw -1, 4, -11, 52, 26,  -8,  3, -1,
+                    dw -1, 3,  -9, 47, 31, -10,  4, -1,
+                    dw -1, 4, -11, 45, 34, -10,  4, -1,
+                    dw  0, 3,   9, 20, 20,   9,  3,  0,
+                    dw -1, 4, -10, 34, 45, -11,  4, -1,
+                    dw -1, 4, -10, 31, 47,  -9,  3, -1,
+                    dw -1, 3,  -8, 26, 52, -11,  4, -1,
+                    dw  0, 1,  -5, 17, 58, -10,  4, -1,
+                    dw  0, 1,  -4, 13, 60,  -8,  3, -1,
+                    dw  0, 1,  -3,  8, 62,  -5,  2, -1,
+                    dw  0, 1,  -2,  4, 63,  -3,  1,  0,
+                
+                    ; 1x, affine, Table 30
+                    dw 0, 0,   0, 64,  0,   0,  0,  0,
+                    dw 0, 1,  -3, 63,  4,  -2,  1,  0,
+                    dw 0, 1,  -5, 62,  8,  -3,  1,  0,
+                    dw 0, 2,  -8, 60, 13,  -4,  1,  0,
+                    dw 0, 3, -10, 58, 17,  -5,  1,  0,
+                    dw 0, 3, -11, 52, 26,  -8,  2,  0,
+                    dw 0, 2,  -9, 47, 31, -10,  3,  0,
+                    dw 0, 3, -11, 45, 34, -10,  3,  0,
+                    dw 0, 3, -11, 40, 40, -11,  3,  0,
+                    dw 0, 3, -10, 34, 45, -11,  3,  0,
+                    dw 0, 3, -10, 31, 47,  -9,  2,  0,
+                    dw 0, 2,  -8, 26, 52, -11,  3,  0,
+                    dw 0, 1,  -5, 17, 58, -10,  3,  0,
+                    dw 0, 1,  -4, 13, 60,  -8,  2,  0,
+                    dw 0, 1,  -3,  8, 62,  -5,  1,  0,
+                    dw 0, 1,  -2,  4, 63,  -3,  1,  0
+
+pw_vvc_iter_shuffle_index_half dw  0,  1,
```

Comment:
maybe we can use %rep like https://github.com/ffvvc/FFmpeg/blob/main/libavcodec/x86/vvc_alf.asm#L31